README.md

```diff
@@ -175,3 +175,50 @@ String yburl = "jdbc:yugabytedb://127.0.0.1:5433/yugabyte?user=yugabyte&password
     Otherwise, when the driver needs to connect to any of these additional endpoints (when the primary endpoint
     specified via `serverName` is unavailable), it will use the default port number (`5433`) and not
     `dataSource.portNumber` to connect to them.
+
+## Logging using java.util.logging
+
+The YugabyteDB JDBC Driver supports the use of logging (or tracing) to help resolve issues with the pgJDBC Driver when is used in your application.
+
+The smart Driver uses the logging APIs of java.util.logging that is part of Java since JDK 1.4, which makes it a good choice for the driver since it doesn’t add any external dependency for a logging framework. java.util.logging is a very rich and powerful tool.
+
+This logging support was added since version 42.7.3-yb-1 of the YugabyteDB pgJDBC Driver, and previous versions uses a custom mechanism to enable logging that it is replaced by the use of java.util.logging in current versions, the old mechanism is no longer available.
+
+### Configuration 
+The Logging APIs offer both static and dynamic configuration control. Static control enables field service staff to set up a particular configuration and then re-launch the application with the new logging settings. Dynamic control allows for updates to the logging configuration within a currently running program.
+
+The root logger used by the pgJDBC driver is `com.yugabyte`.
+
+Enable logging by using logging.properties file 
+The default Java logging framework stores its configuration in a file called logging.properties . Settings are stored per line using a dot notation format. Java installs a global configuration file in the lib folder of the Java installation directory, although you can use a separate configuration file by specifying the java.util.logging.config.file property when starting a Java program. logging.properties files can also be created and stored with individual projects.
+
+The following is an example of setting that you can make in the logging.properties :
```

Comment:
Please explain what this config does. e.g. logs at FINEST level for 5 classes specified at the end and at INFO (or FINEST?) level for rest of them?